polars/polars-core/src/named_from.rs

```diff
@@ -144,6 +144,33 @@ impl<T: AsRef<[Series]>> NamedFrom<T, ListType> for Series {
     }
 }
 
+// #[cfg(feature = "dtype-fixed-size-list")]
```

Comment:
If I uncomment this block then on this line https://github.com/pola-rs/polars/blob/06d58dc62214a88c0f33e44b182ac2c18c612e45/polars/polars-core/src/frame/explode.rs#L387 I get
```rs
error[E0283]: type annotations needed
   --> polars/polars-core/src/frame/explode.rs:387:20
    |
387 |         let list = Series::new("foo", &[s0, s1.clone(), s1.clear()]);
    |                    ^^^^^^^^^^^ cannot infer type for type parameter `Phantom` declared on the trait `NamedFrom`
    |
note: multiple `impl`s satisfying `series::Series: named_from::NamedFrom<&[series::Series; 3], _>` found
   --> polars/polars-core/src/named_from.rs:121:1
    |
121 | impl<T: AsRef<[Series]>> NamedFrom<T, ListType> for Series {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
148 | impl<T: AsRef<[Series]>> NamedFrom<T, FixedSizeListType> for Series {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Is there syntax here for saying I want to use `NamedFrom` to get a `ListType` `Series` and not a `FixedSizeListType` Series?